ui/frontend/build/src/utils.ts

```diff
@@ -0,0 +1,3 @@
+export function useRandomSeed(){
+  return Math.floor(Math.random() * 10000);
```

Comment:
The maximum seed can probably go higher. I've been running it with 2^32 so far and it worked fine.